tests/characters/test_character_control.py

```diff
@@ -83,6 +83,32 @@ def test_alice_character_control_grant(alice_control_test_client, federated_bob)
     assert response.status_code == 400
 
 
+def test_alice_character_control_revoke(alice_control_test_client, federated_bob):
+    bob_pubkey_enc = federated_bob.public_keys(DecryptingPower)
+
+    request_data = {
+        'bob_encrypting_key': bytes(bob_pubkey_enc).hex(),
+        'bob_verifying_key': bytes(federated_bob.stamp).hex(),
+        'label': 'test',
+        'm': 2,
+        'n': 3,
+        'expiration': (maya.now() + datetime.timedelta(days=3)).iso8601(),
+    }
+    response = alice_control_test_client.put('/grant', data=json.dumps(request_data))
+    assert response.status_code == 200
+
+    response_data = json.loads(response.data)
+    policy_pubkey = response_data['result']['policy_encrypting_key']
+
+    response = alice_control_test_client.delete(f'/revoke/{policy_pubkey}')
+    assert response.status_code == 200
+
+    response_data = json.loads(response.data)
+    assert 'result' in response_data
+    assert 'failed_revocations' in response_data['result']
+    assert response_data['result']['failed_revocations'] == 0
```

Comment:
Would be great to have tests with having failed revocations also (because there could be bugs, as mentioned in a comment above). Not necessarily in this PR, but...

Reply:
Opened #879